lmfdb/utils/utilities.py

```diff
@@ -684,7 +684,7 @@ def bigint_knowl(n, cutoff=20, max_width=70, sides=2):
             lng = r"\(%s\)" % lng
         return r'<a title="[bigint]" knowl="dynamic_show" kwargs="%s">\(%s\)</a>'%(lng, short)
     else:
-        return r'\(%s\)'%n
+        return r'%s'%n
```

Comment:
Why make this change?  Don't we still want mathmode even when not including a knowl?